spec.html

```diff
@@ -2444,7 +2444,7 @@ <h1>Object Internal Methods and Internal Slots</h1>
                 (a List of <em>any</em>, Object) <b>&rarr;</b> Object
               </td>
               <td>
-                Creates an object. Invoked via the `new` or `super` operators. The first argument to the internal method is a list containing the arguments of the operator. The second argument is the object to which the `new` operator was initially applied. Objects that implement this internal method are called <em>constructors</em>. A function object is not necessarily a constructor and such non-constructor function objects do not have a [[Construct]] internal method.
+                Creates an object. Invoked via the `new` or `super` expressions. The first argument to the internal method is a list containing the arguments of the expression. The second argument is the object to which the `new` operator was initially applied. Objects that implement this internal method are called <em>constructors</em>. A function object is not necessarily a constructor and such non-constructor function objects do not have a [[Construct]] internal method.
```

Comment:
I’m not sure it makes sense to say an expression has arguments, but i don’t have a better suggestion.

Reply:
Thank you! Another thought of mine is `call`:  “Invoked via the `new` or `super` calls.” and “...a list containing the arguments of the call.”
Which looks better?

Reply:
call to me implies a function, so I’m not sure that makes sense either for new.

Maybe:
```suggestion
                Creates an object. Invoked via the `new` operator or a `super` call. The first argument to the internal method is a list containing the arguments of the constructor invocation or the `super` call. The second argument is the object to which the `new` operator was initially applied. Objects that implement this internal method are called <em>constructors</em>. A function object is not necessarily a constructor and such non-constructor function objects do not have a [[Construct]] internal method.
```

Reply:
Thank you! Yours looks totally fine to me. I will adopt your suggestion. (It seems that GitHub doesn't allow me to adopt a suggestion that is created too recently.)

Reply:
Adopted.